src/components/headerNav.js

```diff
@@ -0,0 +1,102 @@
+
+import React,{ Component } from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { push } from 'react-router-redux'
+import { Route } from 'react-router-dom'
+import AppBar from 'material-ui/AppBar';
+import IconButton from 'material-ui/IconButton';
+import IconMenu from 'material-ui/IconMenu';
+import MenuItem from 'material-ui/MenuItem';
+import FlatButton from 'material-ui/FlatButton';
+
+import Avatar from 'material-ui/Avatar';
+import RaisedButton from 'material-ui/RaisedButton';
+import '../css/index.css'
+
+class HeaderNav extends Component{
+  state = {
+      props:'',
+      logged: false,
+      user:{
+        img:"https://images.duckduckgo.com/iu/?u=https%3A%2F%2Fwww.idyllwildarts.org%2Fwp-content%2Fuploads%2F2016%2F09%2Fblank-profile-picture.jpg&f=1",
```

Comment:
probably store locally

Reply:
Will read from the store as soon as this branch  is merged with develop